src/js/tracks/text-track-display.js

```diff
@@ -290,6 +288,8 @@ class TextTrackDisplay extends Component {
         this.setAttribute('aria-live', 'assertive');
       }
       this.updateForTrack(descriptionsTrack);
+    } else {
+      this.clearDisplay();
```

Comment:
This is necessary to clear the cues whenever we turn off all the captions. This makes it so that this PR is backward and forwards compatible.